optimas/generators/__init__.py

```diff
@@ -32,4 +32,5 @@
     "GridSamplingGenerator",
     "LineSamplingGenerator",
     "RandomSamplingGenerator",
+    "ExternalGenerator",
```

Comment:
Would it make sense to add a test for this object in the CI?
(This could potentially be a follow-up PR.)

Reply:
yes